projects/helper/coreAssets.json

```diff
@@ -37,6 +37,8 @@
     "LUSD": "0x5f98805A4E8be255a32880FDeC7F6728C6568bA0",
     "sUSD_OLD": "0x57ab1e02fee23774580c119740129eac7081e9d3",
     "sUSD": "0x57Ab1ec28D129707052df4dF418D58a2D46d5f51",
+    "OMI": "0xeD35af169aF46a02eE13b9d79Eb57d6D68C1749e",
```

Comment:
for now, I am trying to keep this list short, main reason is, this list sort of acts as base Tokens while pricing some unknown token